aries-site/src/examples/templates/filtering/FilteringLists/FilteringLists.js

```diff
@@ -0,0 +1,139 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { Box, Header, Heading, List, ResponsiveContext, Text } from 'grommet';
+import { Deliver, Package, StatusCritical, StatusGood } from 'grommet-icons';
+
+import {
+  FilterControls,
+  FiltersProvider,
+  useFilters,
+} from '../../FilterControls';
+import { orders } from './mockData';
+
+const statusIcons = {
+  Cancelled: <StatusCritical />,
+  Delivered: <StatusGood />,
+  'In Transit': <Deliver />,
+  'Ready to Ship': <Package />,
+};
+
+export const FilteringLists = ({ containerRef }) => {
+  // containerRef is for demonstration purposes on this site. Most
+  // implementations should likely remove.
+
+  // Define which attributes should be made available for the user
+  // to filter upon
+  const filtersConfig = [
+    {
+      property: 'service',
+      label: 'Service',
+      filterType: 'CheckBoxGroup',
+      // contentProps accept any Grommet Box props to customize the filter's
+      // FormField. https://v2.grommet.io/formfield#contentProps
+      contentProps: { width: 'small' },
+    },
+    {
+      property: 'status',
+      label: 'Status',
+      filterType: 'CheckBoxGroup',
+      contentProps: { width: 'small' },
+    },
+    {
+      property: 'tenant',
+      label: 'Tenant',
+      filterType: 'CheckBoxGroup',
+      contentProps: { width: 'small' },
+    },
+  ];
+
+  // Customize layer properties. Any Grommet Layer props, plus Box props
+  // for the Layer's container and contents.
+  const layerProps = {
+    // containerProps accept any Grommet Box props to customize the Box
+    // containing the layer's header, main content, and footer
+    containerProps: {
+      fill: false,
+      pad: { horizontal: 'medium', vertical: 'small' },
+      width: 'large',
+    },
+    // contentProps accept any Grommet Box props to customize the Box
+    // containing the FormField filters.
+    contentProps: {
+      direction: 'row',
```

Comment:
Think we'll need to use responsive context so this can be aligned in a column on mobile:
<img width="1676" alt="Screen Shot 2021-06-03 at 11 16 27 AM" src="https://user-images.githubusercontent.com/12522275/120741722-6735de00-c4aa-11eb-95cb-1c5400b5e922.png">